src/App.tsx

```diff
@@ -16,46 +54,73 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
+
+  const reorderedGoods = getReorderedGoods(
+    goodsFromServer, { sortType, isReversed },
+  );
+
+  const hasOrderChanged = isReversed || sortType !== SortType.NONE;
+
+  const setClass = (condition: boolean) => {
+    return classNames('button', { 'is-light': condition });
+  };
+
+  const setReset = () => {
```

Comment:
```suggestion
  const reset = () => {
```
It would be better to remove 'set' from simple handler name to not confuse with state